extension/chrome/elements/pgp_block.ts

```diff
@@ -106,7 +113,117 @@ export class PgpBlockView extends View {
+      const from = Str.parseEmail(GmailParser.findHeader(gmailMsg, 'from') ?? '');
+      const fromHtml = from.name ? `<b>${from.name}</b> &lt;${from.email}&gt;` : from.email;
+      const ccString = GmailParser.findHeader(gmailMsg, 'cc') ? `Cc: <span data-test="print-cc">${Xss.escape(GmailParser.findHeader(gmailMsg, 'cc')!)}</span><br/>` : '';
+      const bccString = GmailParser.findHeader(gmailMsg, 'bcc') ? `Bcc: <span>${Xss.escape(GmailParser.findHeader(gmailMsg, 'bcc')!)}</span><br/>` : '';
+      this.printMailInfoHtml = `
+      <hr>
+      <p class="subject-label" data-test="print-subject">${GmailParser.findHeader(gmailMsg, 'subject')}</p>
+      <hr>
+      <br/>
+      <div>
+        <div class="inline-block">
+          <span data-test="print-from">From: ${fromHtml}</span>
+        </div>
+        <div class="float-right">
+          <span>${sentDateStr}</span>
+        </div>
+      </div>
+      <span data-test="print-to">To: ${Xss.escape(GmailParser.findHeader(gmailMsg, 'to') ?? '')}</span><br/>
+      ${ccString}
+      ${bccString}
+      <br/><hr>
+    `;
+    } catch (e) {
+      this.errorModule.debug(`Error while getting gmail message for ${this.msgId} message. ${e}`);
+    }
+  };
+
+  private printPGPBlock = async () => {
+    const w = window.open();
+    const html = `
+      <!DOCTYPE html>
+      <html lang="en-us">
+      <head>
+        <style>
+          #action_show_quoted_content {
+            display: none;
+          }
+          .print_header_info {
+            color: #777777;
+            font-weight: bold;
+            -webkit-print-color-adjust: exact;
+          }
+          .print_encrypted_with_label {
+            display: table-cell;
+            vertical-align: middle;
+            padding-right: 5px;
+          }
+          .subject-label {
+            font-weight: bold;
+            font-size: 20px;
+          }
+          .inline-block {
+            display: inline-block;
+          }
+          .display-table {
+            display: table;
+          }
+          .float-right {
+            float: right;
+          }
+          .quoted_content {
+            display: none;
+          }
+          #attachments a.preview-attachment {
+            display: inline-flex;
+            color: #333 !important;
+            align-items: center;
+            height: 36px;
+            background: #f8f8f8;
+            border: 1px solid #ccc;
+            border-left: 4px solid #31a217 !important;
+            font-family: inherit;
+            font-size: inherit;
+            margin-bottom: 8px;
+            cursor: pointer;
+            margin-right: 12px;
+            padding: 0 8px;
+            text-decoration: none;
+          }
+          #attachments a.preview-attachment .download-attachment {
+            display: none;
+          }
+        </style>
+      </head>
+      <body>
+        ${$('#print-header').html()}
+        <br>
+        ${this.printMailInfoHtml}
```

Comment:
Should sanitize this value here. I think it could potentially contain some security-nasty content, causing HTML injection, maybe.